docs/devel/testing.md

```diff
@@ -165,31 +198,51 @@ is [table driven testing](https://github.com/golang/go/wiki/TableDrivenTests)
 
 ### Install etcd dependency
 
-Kubernetes integration tests require your PATH to include an [etcd](https://github.com/coreos/etcd/releases) installation.
-Kubernetes includes a script to help install etcd on your machine.
+Kubernetes integration tests require your PATH to include an
+[etcd](https://github.com/coreos/etcd/releases) installation. Kubernetes
+includes a script to help install etcd on your machine.
 
 ```sh
 # Install etcd and add to PATH
 
 # Option a) install inside kubernetes root
-cd kubernetes
 hack/install-etcd.sh  # Installs in ./third_party/etcd
```

Comment:
I'll take a PR in a follow-up.  Mostly wanted to get the filtering in for tests now since it is a topic of considerable confusion